cps-wallet-connector/README.md

```diff
@@ -0,0 +1,221 @@
+---
+CPS: 10
+Title: Wallet Connectors
+Status: Open
+Category: Wallets
+Authors:
+    - Ryan Williams <ryan.williams@iohk.io>
+Proposed Solutions: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/619
+Created: 2023-07-28
+---
+
+## Abstract
+Wallets are a foundational element of web 3, being the primary interface between users and blockchains.
+Wallet-web bridges allow users to connect their wallet to web-based stacks, allowing a wide range of specialized experiences.
+
+Wallet web-bridge standards consist of two parts; connection standard and API.
+With the connection standard defining how the wallet and dApp initiate communication, this could be over injected Javascript object, WebRTC, WebTorrent, etc.
```

Comment:
```suggestion
With the connection standard defining how the wallet and dApp initiate communication, for example it could be over injected Javascript object.
```